### ISSUE TITLE
Create Customer

### ISSUE THREAD

**Issue by stefanofalasca** [mariodavid/rtcab-ordermanagement#10]
Hi Mario,
when I try yo create customer :

```
java.lang.IllegalArgumentException: User has currently no tenant assigned. Entity instance cannot be created
	at com.roadtocubaandbeyond.ordermanagement.entity.StandardTenantEntity.initTenantId(StandardTenantEntity.java:39)
```

**Comment by mariodavid** [comment]
Hi,

this happens because I accidentally implemented #3 within #8 . I will upload the video solving #3 in the next days...

Bye
Mario

**Comment by mariodavid** [comment]
done in #3 --> video with explanation: https://youtu.be/s3AFVtHOByU

Bye
Mario
